libraries/tracking/streams/pages.js

```diff
@@ -37,9 +42,16 @@ export const blacklistedPatterns = [
 ];
 
 /**
- * Emits when one of the tracked paths is entered except some special one.
+ * Route did enter and PWA is visible and route is not blacklisted
+ * @type {Rx.Observable<*[]>}
+ */
+const routeDidEnterForVisiblePwa$ = routeDidEnter$
+  .withLatestFrom(appWillStart$.merge(pwaVisibility$))
+  .filter(([, { action: { type } }]) => type === APP_WILL_START || type === PWA_DID_APPEAR)
+  .map(([routeDidEnter]) => routeDidEnter);
```

Comment:
The `startpage` is now tracked twice when the app starts, since both actions are are dispatched in this situation.

![bildschirmfoto 2019-02-15 um 13 52 23](https://user-images.githubusercontent.com/13087792/52858054-61752c80-3129-11e9-8e1f-89172e50bcda.png)